cdap-ui/app/cdap/components/PipelineDetails/PipelineDetailsTopPanel/PipelineDetailsButtons/PipelineRunButton.js

```diff
@@ -61,7 +61,7 @@ export default class PipelineRunButton extends Component {
       } else {
         let { runtimeArgs } = this.props;
         // Arguments with empty values are assumed to be provided from the pipeline
```

Comment:
Should the comment be updated? (Same for the corresponding comment below)

Reply:
The comment still stands true. Previously, instead of ignoring the empty values, we were only adding empty values, that was the bug.